src/Transliterator/data/be/default.php

```diff
@@ -9,6 +9,8 @@
  * file that was distributed with this source code.
  */
 
+namespace Transliterator;
```

Comment:
Why?

Reply:
I think add this to identify the source code is under the ```Transliterator``` namespace.

Reply:
There is no need, is there?

Reply:
Oh, you're right... The namespace is no need there.
The map files in ```data``` folder are require directly.
So I have to remove these namespace in all map files.